guides/source/7_1_release_notes.md

```diff
@@ -65,6 +65,43 @@ TODO: https://github.com/rails/rails/pull/45602
 
 TODO: https://github.com/rails/rails/pull/46049
 
+### Extend `response.parsed_body` to parse HTML with Nokogiri
```

Comment:
@rafaelfranca is this content appropriate for an addition to [the 7.1 release notes](https://discord.com/channels/849034466856665118/974005005768069211/1143220081002823781)?

Reply:
Would this be better suited as two separate PRs?

1. Add Release Notes for the original work done in https://github.com/rails/rails/pull/47144
1. Add test coverage for pattern matching, then expand on the Release Notes

Reply:
I'll review this list later, since this is a highlight sections, but I prefer to have more and remove than have fewer things and trying to chase in more than 1000s commits